PyPDF2/_merger.py

```diff
@@ -669,7 +675,13 @@ def addBookmark(
         """
         deprecate_with_replacement("addBookmark", "add_outline_item")
         return self.add_outline_item(
-            title, pagenum, parent, color, bold, italic, fit, *args
+            title,
+            pagenum,
+            parent,
```

Comment:
If we change the arguments, what about adding before parameter at the same time ?

Reply:
In the same release, but not the same pr :-)